### PR TITLE
Update README to use Fishjam (rebranding)

### PR DESCRIPTION
**Pull request by karkakol**
Update README file to use new name - Fishjam

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,12 +106,12 @@ room_api = RoomApi(server_address='localhost:5002')
 # that allow to use HLS.
 address, room = room_api.create_room(video_codec="h264")
 
-# Create new room api with returned jellyfish address as a room could be
-# created on a different jellyfish instance
-# (if you communicate with a cluster of jellyfishes)
+# Create new room api with returned fishjam address as a room could be
+# created on a different fishjam instance
+# (if you communicate with a cluster of fishjames)
```
Comment (Rados13) [inline README.md:111]: We should decide whether the plural form is `fishjam` or `fishjames`.

File: README.md
Comment (Rados13) [inline README.md:1]: I think line 24 also should be modified.

**Review by Rados13 — COMMENTED** [review]
This PR updates only README. Will other things be modified in the following PRs?

File: README.md (hunk 1 above)
Comment (roznawsk) [reply README.md:111]: fishjams definitely

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Jellyfish Python Server SDK
+# Fishjam Python Server SDK
 
 [![CircleCI](https://dl.circleci.com/status-badge/img/gh/jellyfish-dev/python-server-sdk/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/jellyfish-dev/python-server-sdk/tree/main)
```
Comment (roznawsk) [inline README.md:3]: We should update the badges as well

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
-Python server SDK for the [Jellyfish Media Server](https://github.com/jellyfish-dev/jellyfish).
+Python server SDK for the [Fishjam Media Server](https://github.com/jellyfish-dev/jellyfish).
 
 Read the docs [here](https://jellyfish-dev.github.io/python-server-sdk)
```
Comment (roznawsk) [inline README.md:7]: Update the docs link

File: README.md [hunk 4]
```diff
@@ -53,9 +53,9 @@ All methods in `RoomApi` may raise one of the exceptions deriving from `jellyfis
 
 #### Notifier
 
-Notifier allows for receiving real-time updates from the Jellyfish Server.
+Notifier allows for receiving real-time updates from the Fishjam Server.
 
-You can read more about notifications in the [Jellyfish Docs](https://jellyfish-dev.github.io/jellyfish-docs/next/getting_started/notifications).
+You can read more about notifications in the [Fishjam Docs](https://jellyfish-dev.github.io/jellyfish-docs/next/getting_started/notifications).
```
Comment (roznawsk) [inline README.md:58]: update link

**Review by roznawsk — APPROVED** [review]

**Review by Rados13 — APPROVED** [review]

**Pull request merged**